client/layout/masterbar/logged-in.jsx

```diff
@@ -525,7 +527,10 @@ class MasterbarLoggedIn extends Component {
 
 		return (
 			<Item
-				className="masterbar__item-launch-site"
+				as={ Button }
+				variant="primary"
+				// Keep the Launch button always in blueberry (default scheme: modern) like in wp-admin.
+				className={ clsx( 'masterbar__item-launch-site', 'color-scheme', 'is-global' ) }
```

Comment:
I was wondering how this should be done, setting color-scheme classes as a way to run the override for everything below wfm. :+1:

Reply:
Works on the notifications panel too https://github.com/Automattic/wp-calypso/pull/101708